src/intervals/set_operations.jl

```diff
@@ -119,6 +119,8 @@ hull(a::Interval, b::Interval) = Interval(min(a.lo, b.lo), max(a.hi, b.hi))
 # hull{T,S}(a::Interval{T}, b::Interval{S}) = hull(promote(a, b)...)
 hull(a::Complex{<:Interval},b::Complex{<:Interval}) =
     complex(hull(real(a),real(b)),hull(imag(a),imag(b)))
+hull(a, b...) = reduce(hull, (a, b...))
```

Comment:
You can write this as 
```jl
hull(a...) = reduce(hull, a)
```

Reply:
Thanks! I just pushed a new commit following your suggestion.